### ISSUE TITLE
Modify model zoo UI

### ISSUE THREAD

**Issue by utsavgarg** [Cloud-CV/Fabrik#311]
The model zoo currently opens up as a modal, and therefore cannot hold all models that have been [tested with Fabrik](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/tested_models.md) as it will get too cluttered. Even without all the models present, there are many models which aren't as popular and it might be confusing for new users who might not be familiar with them.

Therefore, it might be better to initially show only popular models(which have well-cited research papers) in the zoo and have a *load more* button at the bottom which reveals all the remaining models.

The task has the following deliverables:
- [ ] Add all models from [tested models](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/tested_models.md) to the model zoo
- [ ] Only show popular models initially
- [ ] Reveal all remaining models on clicking the *load more* button

Extended functionality:
- [ ] Redirect to a new page when the user clicks *load more*, the new page has a search bar at the top
- [ ] The new page should have models displayed in card view each with a preview of the architecture

**Comment by thatbrguy** [comment]
@utsavgarg With reference to an old issue #300, the links at tested_models.md needs to be replaced with newer share links, to fix a bug related to parameter count and keras export. I would like to suggest that this be added as a sub-issue for anyone who wants to take it.

**Comment by MuLx10** [comment]
I want to take up this issue.I am currently working on it

**Comment by laylalaisy** [comment]
Do you think it's proper and better if we add an search bar at the top of the model ZOO UI,  so the user can search the wanted model quicker? @utsavgarg

**Comment by utsavgarg** [comment]
@laylalaisy Yes, that is a good idea. We can include a search bar which dynamically filters the list of models.

**Comment by laylalaisy** [comment]
@utsavgarg Thx and I would like to implement it! I think search bar can be more convenient for some users.

**Comment by utsavgarg** [comment]
@laylalaisy it's a nice effort, but the UI doesn't seem to scale well with different screen sizes. Also, I have updated the feature description, see if you can incorporate those changes in this PR.

![image](https://user-images.githubusercontent.com/11680179/37117377-03e2b422-228c-11e8-9992-77665e4271d3.png)

**Comment by utsavgarg** [comment]
@MuLx10 @laylalaisy Instead of both of you working on this, it would be better for either one of you to take the issue or split the features so that no effort is wasted.

**Comment by MuLx10** [comment]
@utsavgarg I'll incorporate the extensions and discuss the split up with @laylalaisy
